Sources/TuistScaffold/TemplateGenerator.swift

```diff
@@ -115,10 +116,11 @@ public final class TemplateGenerator: TemplateGenerating {
                     context: attributes
                 )
             case let .file(path):
+                let injectedLoaderEnvironment = stencilSwiftEnvironment(templatePaths: [Path(path.dirname)])
                 let fileContents = try FileHandler.shared.readTextFile(path)
                 // Render only files with .stencil extension
                 if path.extension == "stencil" {
-                    renderedContents = try environment.renderTemplate(
+                    renderedContents = try injectedLoaderEnvironment.renderTemplate(
```

Comment:
Is this change related to `fail to load stencils using {% extends %}`? Can we split the 2 fixes in 2 PRs so we can have 2 changelog entries?

Reply:
Thanks 🚀